build/entities.js

```diff
@@ -19,6 +19,21 @@ class StartTestItem {
             const attrs = tags.map((value) => (new ReporterOptions_1.Attribute("tag", value)));
             this.attributes.push(new ReporterOptions_1.Attribute("func", tags[0]));
             this.attributes.push(...attrs);
+            // if tags are detected, it's either the name of test feature, like:
+            // @tag1 @tag2 Test for a feature
+            //
+            // or the actual test scenario, like:
+            // @tag3, @tag4: test for a specific senceario
+            //
+            // on these 2 cases, we parse out the test name
+            // and send it over to report portal as attribute so each test feture / scenario
+            // can be uniquely identified via the attribute to create dashboard that
+            // can drilled down to individual test case level
+            const testName = this.name.match(/: (.+$)/);
+            if (testName && testName.length > 1) {
+                const title = testName[1].replace(/ /g, '-');
+                this.attributes.push(new ReporterOptions_1.Attribute("title", title));
```

Comment:
actually I didn't alter the reporter logic that much (more like patching it in this regard); the mentioned logic seems to remove the tags from the test title while logging to the report portal under an option param, and my codes will log the test title as an attribute regardless of this param.

I prefer to not touch this part of code in case it introduces side effort of the overall logging logic....

Reply:
create helper function as suggested now!

Reply:
just a note: this project also tracks compiled codes from TS; so changes to JS are actually programmatically generated.

Reply:
Thanks! I was wondering if they might be auto-generated! :D